VisualizationBase/src/items/ViewItem.h

```diff
@@ -65,7 +65,8 @@ class VISUALIZATIONBASE_API ViewItem : public Super<DeclarativeItem<ViewItem>> {
 		QList<QPair<Item*, Item*>> arrowsForLayer(QString layer);
 		QString fullLayerName(QString localLayer);
 
-		const QString name() const;
+		QString name() const;
+		void setName(QString name);
```

Comment:
Also take a `const QString&`. In general that's a good way to pass around QString stuff.